custom/icds_reports/utils/data_pull_scripts/Oct_data_pull_segregation_data.sql

```diff
@@ -0,0 +1,169 @@
+-- FETCH BY SUPERVISOR AND BRING TO MASTER, SIMILAR QUERY Needs to performed on CCS RECORD
+create unlogged table temp_thr_data_pull as select
+supervisor_id,
+SUM(CASE WHEN num_rations_distributed is not null and num_rations_distributed=0 then 1 ELSE 0 END) as child_thr_0,
+SUM(CASE WHEN num_rations_distributed is not null and num_rations_distributed>0 and num_rations_distributed<=7 then 1 ELSE 0 END) as child_thr_1_7,
+SUM(CASE WHEN num_rations_distributed is not null and num_rations_distributed>=8 and num_rations_distributed<=14 then 1 ELSE 0 END)as  child_thr_8_14,
+SUM(CASE WHEN num_rations_distributed is not null and num_rations_distributed>=15 and num_rations_distributed<=21 then 1 ELSE 0 END)as child_thr_15_21,
+SUM(CASE WHEN num_rations_distributed is not null and num_rations_distributed>21 then 1 ELSE 0 END) as child_thr_gt_21,
+SUM(thr_eligible) as thr_eligible
+from "child_health_monthly" child_health where month='2019-10-01'
+group by supervisor_id;
+​/*
+ HashAggregate  (cost=0.00..0.00 rows=0 width=0)
+   Group Key: remote_scan.supervisor_id
+   ->  Custom Scan (Citus Real-Time)  (cost=0.00..0.00 rows=0 width=0)
+         Task Count: 64
+         Tasks Shown: One of 64
+         ->  Task
+               Node: host=100.71.184.232 port=5432 dbname=icds_ucr
+               ->  Finalize GroupAggregate  (cost=1000.64..417186.39 rows=306 width=81)
+                     Group Key: supervisor_id
+                     ->  Gather Merge  (cost=1000.64..417156.55 rows=1530 width=81)
+                           Workers Planned: 5
+                           ->  Partial GroupAggregate  (cost=0.56..415972.23 rows=306 width=81)
+                                 Group Key: supervisor_id
+                                 ->  Parallel Index Scan using chm_month_supervisor_id_102648 on child_health_monthly_102648 child_health  (cost=0.56..411701.07 rows=113816 width=41)
+                                       Index Cond: (month = '2019-10-01'::date)
+(15 rows)
+
+*/
+
+create unlogged table temp_pse_data_pull as select
+supervisor_id,
+SUM(CASE WHEN pse_days_attended is not null and pse_days_attended=0 then 1 ELSE 0 END) as child_pse_0,
+SUM(CASE WHEN pse_days_attended is not null and pse_days_attended>0 and pse_days_attended<=7 then 1 ELSE 0 END) as child_pse_1_7,
+SUM(CASE WHEN pse_days_attended is not null and pse_days_attended>=8 and pse_days_attended<=14 then 1 ELSE 0 END)as  child_pse_8_14,
+SUM(CASE WHEN pse_days_attended is not null and pse_days_attended>=15 and pse_days_attended<=21 then 1 ELSE 0 END)as child_pse_15_21,
+SUM(CASE WHEN pse_days_attended is not null and pse_days_attended>21 then 1 ELSE 0 END) as child_pse_gt_21,
+SUM(pse_eligible) as pse_eligible
+from "child_health_monthly" child_health where month='2019-10-01' and age_tranche::INTEGER  BETWEEN 37 AND 72
```

Comment:
Is there a concern that a child could have a value in `pse_days_attended` and `age_tranche::INTEGER  BETWEEN 37 AND 72` is false? If having a value in pse_days_attended requires this condition, then this could easily be added to the query above